packages/chat-component-bindings/src/utils/updateMessagesWithAttached.ts

```diff
@@ -26,8 +26,16 @@ export const updateMessagesWithAttached = (chatMessagesWithStatus: Message[]): v
     const previousSenderId = previousMessage?.messageType === 'chat' ? previousMessage.senderId : undefined;
     const nextSenderId = nextMessage?.messageType === 'chat' ? nextMessage.senderId : undefined;
 
+    const timediff =
+      new Date(message?.createdOn ?? '').getTime() - new Date(previousMessage?.createdOn ?? '').getTime();
```

Comment:
I believe `new Date('')` will end up creating a new date time object with the latest date.
If `createdOn` properties are ever blank, this would fail.
Can you try taking this into account in the tests you wrote?

Ideally, this logic shouldn't run if createdOn date is not present for the previous message or the current message.

Reply:
Yes I tested this scenario, if createdOn is empty, new Date(' ') returns an invalid date object and in this case timediff will be NaN, then it wont meet the logic if (timediff && timediff >=5)

Reply:
Updated test :)